content/docs/manage/projects.md

```diff
@@ -158,6 +159,104 @@ To configure the history retention period for a project:
 4. Use the slider to select the history retention period.
 5. Click **Save**.
 
+### Configure IP Allow
+
+Available to [Neon Pro Plan](/docs/introduction/pro-plan) users, the IP Allow feature provides an added layer of security for your data, restricting access to the branch where your database resides to only those IP addresses that you specify. In Neon, the IP allowlist is applied to all branches by default. Optionally, you can apply the IP allowlist to your project's [primary branch](/docs/manage/branches#primary-branch) only. For instance, you might want to restrict access to the primary branch to a handful of trusted IPs while allowing broader access to development branches.
+
+By default, Neon allows IP addresses from 0.0.0.0, which means that Neon accepts connections from any IP address. Once you configure IP Allow by adding IP addresses or ranges, only those IP addresses will be allowed to access to Neon.
+
+<Admonition type="note">
+Currently, Neon only supports [IPv4](https://en.wikipedia.org/wiki/Internet_Protocol_version_4).
+</Admonition>
+
+You configure **IP Allow** for a project using the Neon Console, CLI, or API:
+
+<Tabs labels={["Neon Console", "CLI", "API"]}>
+
+<TabItem>
+
+1. Select a project in the Neon console.
+2. On the Neon **Dashboard**, select **Settings**.
+3. Select **IP Allow**.
+    ![IP Allow configuration](/docs/manage/ip_allow.png)
+4. Specify the IP addresses you want to permit.
+5. Optionally, select **Apply to primary branch only** to apply the allowlist to your project's [primary branch](/docs/manage/branches#primary-branch) only
+5. Click **Apply changes**.
+
+</TabItem>
+
+<TabItem>
+
+The [Neon CLI projects command](https://neon.tech/docs/reference/cli-projects) supports IP Allow configuration. For example, this command configures IP for an existing Neon project:
+
+```bash
+neonctl projects update still-bird-22384887 --ip-allow 127.0.0.1 192.168.2.1/24 --ip-primary-only
```

Comment:
@Shridhad, did we release it?

Reply:
No, we did not. We are still holding the release as we have one discussion ongoing -  https://neondb.slack.com/archives/C036U0GRMRB/p1702912927587379 
cc: @lpetkov